doc/cephadm/services/index.rst

```diff
@@ -81,6 +81,13 @@ system name:
 
     ceph orch ps --daemon_type osd --daemon_id 0
 
+.. note::
+   The information shown by ``ceph orch ps`` may not be up to date. Daemon status is refreshed every 10 minutes by default.
+   This can be changed by setting the configuration variable ``mgr/cephadm/daemon_cache_timeout`` (in seconds) e.g:
+   ``ceph config set mgr mgr/cephadm/daemon_cache_timeout 60`` will reduce the refresh interval to one minute. In general
+   ``REFRESHED`` will indicate how old is the information shown by ceph orch ps and similar commands. Normally the information
```

Comment:
Suggest backticks around the second "ceph orch ps" on line 88.  I'm a markup junkie.

Reply:
fixed :+1: